modules/tensor_mechanics/src/materials/ComputeCappedWeakPlaneStress.C

```diff
@@ -176,7 +176,7 @@ ComputeCappedWeakPlaneStress::returnMap()
   Real p_ok = _stress_old[_qp](2, 2);
   Real q_ok = std::sqrt(Utility::pow<2>(_stress[_qp](2, 0)) + Utility::pow<2>(_stress[_qp](2, 1)));
   // note that _intnl[_qp] is the "running" intnl variable that gets updated every NR iteration
-  std::array<Real, 2> intnl_ok = {_intnl_old[_qp][0], _intnl_old[_qp][1]};
+  std::array<Real, 2> intnl_ok = {{ _intnl_old[_qp][0], _intnl_old[_qp][1] }};
```

Comment:
I don't see any downsides to sticking with a single pair other than we'd want to remove the warning flag from the various compilers. Maybe we should consider that before this double brace is widespread.